packages/test/src/helpers-integration.ts

```diff
@@ -89,12 +84,22 @@ export async function createTestWorkflowBundle({
   });
 }
 
+export const defaultSearchAttributes = [
```

Comment:
Don't we need those typed SA keys anywhere else in our tests? I mean, they should probably be individually exposed as `const`, or all of them as properties on a `const object`.

Reply:
Could use them in some tests in `test-schedules.ts`, changed to an object